src/bindings/config.ts

```diff
@@ -85,7 +85,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
   }
 
   if (botConfig.payout.privateKey == "") {
```

Comment:
This is not according to the specification of the issue. You need to remove this check here and add it in `src/handlers/payout/action.ts` just like the other checks

Reply:
@whilefoo sorry about that, I have made the requested changes hope this will fix the issue!